src/resolution.js

```diff
@@ -79,32 +109,31 @@ var ri = module.exports = {
 	* @public
 	*/
 	getScreenType: function (rez) {
-		rez = rez || {
-			height: global.innerHeight,
-			width: global.innerWidth
+		rez = rez || _workspaceBounds || {
```

Comment:
Is there something that clears `_workspaceBounds` so that the fallback would be needed?

Reply:
I don't believe there is anything that clears it, so we can probably remove this extra security.